CHANGELOG.md

```diff
@@ -2,6 +2,24 @@
 
 All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
 
+## [1.101.0](https://github.com/rudderlabs/rudder-config-schema/compare/v1.100.0...v1.101.0) (2025-01-14)
```

Comment:
Move all these changes to 1.100.0 version changelog
And add Klaviyo related default change log under 1.101.0

Reply:
updated the changelog @krishna2020